data/phi1017/phi011/__cts__.xml

```diff
@@ -1,11 +1,13 @@
-<cts:work xmlns:cts="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi1017.phi011" groupUrn="urn:cts:latinLit:phi1017" xml:lang="lat">
-  <cts:title xml:lang="lat">Apocolocyntosis</cts:title>
-  <cts:edition urn="urn:cts:latinLit:phi1017.phi011.perseus-lat2" workUrn="urn:cts:latinLit:phi1017.phi011">
-    <cts:label xml:lang="eng">Apocolocyntosis, Petronius, with an English translation by Michael Heseltine Seneca. Apocolocyntosis, with an English translation by W.H.D. Rouse</cts:label>
-    <cts:description xml:lang="eng">Seneca, Lucius Annaeus, ca. 4 B.C.-65 A.D, creator; Petronius Arbiter, creator; Rouse, W. H. D. (William Henry Denham), 1863-1950, editor; Heseltine, Michael, translator; Rouse, W. H. D. (William Henry Denham), 1863-1950, translator</cts:description>
-  </cts:edition>
-  <cts:translation urn="urn:cts:latinLit:phi1017.phi011.perseus-eng2" workUrn="urn:cts:latinLit:phi1017.phi011" xml:lang="eng">
-    <cts:label xml:lang="eng">Apocolocyntosis, Petronius, with an English translation by Michael Heseltine Seneca. Apocolocyntosis, with an English translation by W.H.D. Rouse</cts:label>
-    <cts:description xml:lang="eng">Seneca, Lucius Annaeus, ca. 4 B.C.-65 A.D, creator; Petronius Arbiter, creator; Rouse, W. H. D. (William Henry Denham), 1863-1950, editor; Heseltine, Michael, translator; Rouse, W. H. D. (William Henry Denham), 1863-1950, translator</cts:description>
-  </cts:translation>
-</cts:work>
\ No newline at end of file
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi1017.phi011" groupUrn="urn:cts:latinLit:phi1017" xml:lang="lat">
+  <ti:title xml:lang="lat">Apocolocyntosis</ti:title>
+  <ti:edition urn="urn:cts:latinLit:phi1017.phi011.perseus-lat2" workUrn="urn:cts:latinLit:phi1017.phi011">
+    <ti:label xml:lang="lat">Apocolocyntosis</ti:label>
+    <ti:description xml:lang="mul">Seneca, Lucius Annaeus. Petronius. Seneca Apocolocyntosis. Heseltine, Michael, editor. Rouse, W. H. D. (William Henry Denham), editor. 
```

Comment:
Rouse, W. H. D. (William Henry Denham)

Reply:
hi @lcerrato sorry if I'm confused here, but it looks like Rouse appears exactly in this cts_xml file like you have it spelled out.

Reply:
I thought we were spelling it out and eliminating parentheses?

Reply:
Either Rouse, W. H. D.  or Rouse, William Henry Denham but not both?

Reply:
@ gotcha, I think I will do: Rouse, William Henry Denham then and skip parentheses.